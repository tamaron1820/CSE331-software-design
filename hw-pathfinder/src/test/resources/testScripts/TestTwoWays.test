# Testing whether Dijkstra's algorithm works correctly
# if there are two ways from some node to node which is same cost

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

AddEdge graph1 n1 n2 2.0
AddEdge graph1 n1 n3 5.0
AddEdge graph1 n2 n4 8.0
AddEdge graph1 n3 n4 5.0

FindPath graph1 n1 n4