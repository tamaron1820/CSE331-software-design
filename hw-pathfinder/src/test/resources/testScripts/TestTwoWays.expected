# Testing whether Dijkstra's algorithm works correctly
# if there are two ways from some node to node which is same cost

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

added edge 2.000 from n1 to n2 in graph1
added edge 5.000 from n1 to n3 in graph1
added edge 8.000 from n2 to n4 in graph1
added edge 5.000 from n3 to n4 in graph1

path from n1 to n4:
n1 to n3 with weight 5.000
n3 to n4 with weight 5.000
total cost: 10.000
